rules/coding-style/src/Node/NameImporter.php

```diff
@@ -88,6 +90,10 @@ public function importName(Name $name): ?Name
             return null;
         }
 
+        if ($this->isShortNameInUseStatement($name)) {
```

Comment:
This should be added to ClassNameImportSkipper class, as one of skippers

Reply:
implemented.